src/collections/members/hersh-dhillion/index.mdx

```diff
@@ -9,4 +9,7 @@ location: New Delhi, India
 bio: I am a CS graduate from IIT Ropar currently working as a Software Engineer in the Platforms and Infrastructure team at Flipkart. My work interests lie in building large scale systems and DevOps. Aside from work you'll generally find me analysing data of football (soccer) teams and fantasy sports leagues or reading philosophy. I love the open culture and people here in the Layer5 community. Hit me up if you wanna chat tech, football, philosophy or all things intersectional.
 status: Active
 published: true
+badges:
+  - meshery
+  - smp
```

Comment:
Move it above the `published` frontmater.
`published` should be at last.

Reply:
Sure... Will do